pkgs/applications/emulators/wibo/default.nix

```
@@ -9,13 +9,13 @@
 
 stdenv.mkDerivation rec {
   pname = "wibo";
-  version = "0.4.2";
+  version = "0.6.12";
```

```suggestion
  version = "0.6.16";
```
Latest version by now

---

Thanks, updated